src/applications/static-pages/translation.js

```diff
@@ -10,17 +10,24 @@ const faqI18Content = {
     langToggleLink: '/coronavirus-veteran-frequently-asked-questions-esp',
   },
 };
-
+const configureBreadCrumbLinks = currentLang => {
+  const breadCrumbLinks = document.getElementsByClassName('breadcrumb-link');
+  for (let i = 0; i < breadCrumbLinks.length; i++) {
+    breadCrumbLinks[i].lang = currentLang;
+  }
+};
 const configureTranslationLink = (e, currentLang, targetLang) => {
-  const contentDiv = document.getElementById('content');
-  contentDiv.lang = currentLang;
-  e.dataset.lang = currentLang;
+  e.dataset.lang = targetLang;
+  e.lang = targetLang;
   e.innerText = faqI18Content[targetLang].linkTitle;
   e.href = faqI18Content[targetLang].langToggleLink;
   e.onclick = recordEvent({
-    event: 'faq-lang-toggle',
+    event: 'faq-lang-toggle-click',
+    faqText: faqI18Content[targetLang].title,
```

Comment:
I don’t think this captures a link click. It just logs a GA event on page load I think. It should be wrapped in a function.

Reply:
Oops, didn't see Nick's comment until now 😂 sorry for repeating! https://github.com/department-of-veterans-affairs/vets-website/pull/15710/files#r561233805

Reply:
done!